Detectors/MUON/MCH/Workflow/src/DataDecoderSpec.cxx

```diff
@@ -132,17 +137,20 @@ class DataDecoderTask
       auto cruId = o2::raw::RDHUtils::getCRUID(rdhPtr);
       auto endpoint = o2::raw::RDHUtils::getEndPointID(rdhPtr);
       o2::raw::RDHUtils::setFEEID(rdhPtr, cruId * 2 + endpoint);
```

Comment:
This modifies the O2 message payload in the shared memory. Is this really needed ? From the code it is not obvious.

Reply:
I'll let @aferrero2707 confirm but this is currently needed as the raw data taken so far by mch (maybe not the latest latest with RDHv6) does not have the expected FEEID field correctly set. And the feeId value is central piece of our electronic mapping so it must be correct for the decoder to behave correctly.
This "patchPage" part will be removed eventually.

Reply:
@shahor02 I confirm what @aphecetche said: the patching of the RDH is only needed now as long as we cannot properly set the FEEID when configuring the CRU. We will probably keep a similar function in order to store in memory some of the RDH fields, but there will be no need to change the contents of the memory.